requirements/requirements-ci.txt

```diff
@@ -123,27 +120,27 @@ pycryptodome==3.8.2
 pyflakes==2.1.1
 pygments==2.4.2
 pyhamcrest==1.9.0
-pyinstaller==3.4
+pyinstaller==3.5
 pylint==2.3.1
 pymacaroons-pynacl==0.9.3
 pynacl==1.3.0
 pyopenssl==19.0.0
-pyparsing==2.4.0
-pyrsistent==0.15.2
-pysaml2==4.7.0
+pyparsing==2.4.1.1
+pyrsistent==0.15.3
+pysaml2==4.8.0
 pysha3==1.0.2
 pytest-forked==1.0.2
 pytest-random==0.2
 pytest-select==0.1.2
-pytest-xdist==1.28.0
-pytest==4.6.3
+pytest-xdist==1.29.0
+pytest==5.0.1
```

Comment:
Please refrain from making such unrelated changes inside such a big PR. You are switching us to a new major pytest release. Why introduce this change in this PR?

Reply:
I didn't know `requirements/dep upgrade` can take a package name.  I'll remove unrelated changes in this PR.